standard/types.md

```diff
@@ -6,8 +6,8 @@ The types of the C# language are divided into two main categories: ***reference
 
 ```ANTLR
 type
-    : reference_type
-    | value_type
+    : value_type
```

Comment:
Is this ordering change meaningful? (I'm guessing at one point it was more than this, and then we removed bits? Or is it genuinely meaningful due to precedence of some kind?)